src/utils/binning-helpers.js

```diff
@@ -62,15 +66,13 @@ export function autoBinParams(timeBounds, maxNumBins, reverse) {
     return DEFAULT_NULL_TIME_RANGE
   }
   const timeSpans = reverse ? TIME_SPANS.slice().reverse() : TIME_SPANS
+
   for (let s = 0; s < timeSpans.length; s++) {
-    if (
-      timeRange / timeSpans[s].numSeconds < maxNumBins &&
-      timeRange / timeSpans[s].numSeconds > 2
-    ) {
+    if (timeRange / timeSpans[s].numSeconds < maxNumBins) {
```

Comment:
See [this PR](https://github.com/omnisci/mapd-charting/pull/352) for this specific change